protocols/jabber/jabber.c

```diff
@@ -685,4 +694,9 @@ void jabber_initmodule()
 	ret->buddy_action = jabber_buddy_action;
 
 	register_protocol(ret);
+
+	/* Another one for hipchat, which has completely different logins */
```

Comment:
The only thing you're changing is the name property though?

But sure, copying a prpl is the way to create a new protocol.